src/adafruit_circuitplayground/test/test_pixel.py

```diff
@@ -0,0 +1,111 @@
+import pytest
+
+from ..pixel import Pixel
+
+
+class TestPixel(object):
+
+    def setup_method(self):
+        self.pixel = Pixel(
+            {
+                'brightness': 1.0,
+                'button_a': False,
+                'button_b': False,
+                'pixels': [(255, 0, 0), (0, 255, 0), (0, 0, 255)],
+                'red_led': False,
+                'switch': False
+            })
+
+    @pytest.mark.parametrize("debug_mode", [True, False])
+    def test_set_debug_mode(self, debug_mode):
```

Comment:
Probably out of the scope of this PR, but it would be useful for contributors to have instructions on running the python tests and setting up the environment for the tests.

Reply:
I can actually add this real quick